src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TABDIMS

```diff
@@ -6,128 +6,152 @@
   "size": 1,
   "items": [
     {
+      "item": 1,
       "name": "NTSFUN",
       "value_type": "INT",
       "default": 1
     },
     {
+      "item": 2,
       "name": "NTPVT",
       "value_type": "INT",
       "default": 1
     },
     {
+      "item": 3,
       "name": "NSSFUN",
       "value_type": "INT",
       "default": 20
     },
     {
+      "item": 4,
       "name": "NPPVT",
       "value_type": "INT",
       "default": 20
     },
     {
+      "item": 5,
       "name": "NTFIP",
       "value_type": "INT",
       "default": 1
     },
     {
+      "item": 6,
       "name": "NRPVT",
       "value_type": "INT",
       "default": 20
     },
     {
+      "item": 7,
       "name": "MAX_RV_NODES",
       "value_type": "INT"
+      "default": 20,
+      "comment": "This item is ECLIPSE300 specific"
```

Comment:
You need to append a comma to the `"value_type"` line when inserting new elements into this array.  In other words, this needs to say
```json
{
  "item": 7,
  "name": "MAX_RV_NODES",
  "value_type": "INT",
  "default": 20,
  "comment": "This item is ECLIPSE300 specific"
}
```